cost/azure/unused_sql_databases/azure_unused_sql_databases.pt

```diff
@@ -223,10 +234,10 @@ policy "policy_azure_db_utilization" do
     detail_template <<-EOS
 # Azure Unused SQL Databases
 
-| Region | Name | Resource Group | Kind | Type | SKU - Name | SKU - Tier | SKU - Capacity |
-| ------ | ---- | -------------- | ---- | ---- | -----------| ---------- | -------------- |
+| Subscription Name | Region | Name | Resource Group | Kind | Type | SKU - Name | SKU - Tier | SKU - Capacity |
```

Comment:
please add sorting by subscription name and region

Reply:
Sorted by subscriptionName and location. Also, changed the field name from Region to Location in reporting part as we are fetching the value by .location